src/Tribe/Credits.php

```diff
@@ -23,7 +23,12 @@ public function hook() {
 	 * @return void
 	 **/
 	public function html_comment_credit( $after_html ) {
-		$html_credit = "\n<!--\n" . esc_html__( 'This calendar is powered by The Events Calendar.', 'tribe-common' ) . "\nhttp://m.tri.be/18wn\n-->\n";
+
+		if ( ! class_exists( 'Tribe__Events__Main' ) ) {
+			return;
```

Comment:
return `$after_html`, to prevent breaking if anything was applied to that filter before.